### PR TITLE
COMMERCE-9650 Fixed default order validator

### PR DESCRIPTION
**Pull request by alessiorendina**
Waiting for https://github.com/liferay-commerce/liferay-portal/pull/3049 to be merged first.

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/DefaultCommerceOrderValidatorImpl.java [hunk 1]
```diff
@@ -78,12 +78,8 @@ public CommerceOrderValidatorResult validate(
 			cpInstance);
 		int maxOrderQuantity = cpDefinitionInventoryEngine.getMaxOrderQuantity(
 			cpInstance);
-		String[] allowedOrderQuantities =
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/DefaultCommerceOrderValidatorImpl.java:81]: why is the back order check removed?

Comment (alessiorendina) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/DefaultCommerceOrderValidatorImpl.java:81]: Source formatter complained about it

Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/DefaultCommerceOrderValidatorImpl.java:81]: Then why you didn't remove this check also in the other validate method of this class?

Comment (alessiorendina) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/DefaultCommerceOrderValidatorImpl.java:81]: Yes, fixed. Thanks!

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce

**Comment by lorenzolapescara** [comment]
@alessiorendina I've re-sent at #3077. Please close this PR.
Thanks, Lorenzo.

**Pull request closed without merging**